### PR TITLE
Normalize Timely Vote Credits in Steward score

### PR DESCRIPTION
**Pull request by ebatsell**
Normalizes vote credits which are now 8x higher for validators starting in epoch 704.

### CODE REVIEW

File: programs/validator-history/src/state.rs [hunk 1]
```diff
@@ -274,10 +275,49 @@ impl CircBuf {
         field_latest!(self, epoch_credits)
     }
 
+    /// Normalized epoch credits, accounting for Timely Vote Credits making the max number of credits 8x higher
+    /// for every epoch starting at `tvc_activation_epoch`
```
Comment (ebatsell) [inline programs/validator-history/src/state.rs:279]: add a comment listing activation epochs

File: programs/steward/src/score.rs [hunk 2]
```diff
@@ -90,6 +90,7 @@ pub fn validator_score(
     cluster: &ClusterHistory,
     config: &Config,
     current_epoch: u16,
```
Comment (coachchucksol) [inline programs/steward/src/score.rs:92]: current_epoch should be u64

**Review by coachchucksol — APPROVED** [review]
lgtm

File: programs/steward/src/score.rs (hunk 2 above)
Comment (ebatsell) [reply programs/steward/src/score.rs:92]: legacy issue cause of ValidatorHistoryEntry trying to minimize bytes

**Pull request merged**